src/state/user-atom.ts

```diff
@@ -0,0 +1,8 @@
+import { UserBase } from "@/lib/types/common";
+import { atomWithStorage, createJSONStorage } from "jotai/utils";
+
+export const userAtom = atomWithStorage<UserBase | undefined>(
```

Comment:
let's make it clear that this is auth user.

```suggestion
export const authUserAtom = atomWithStorage<UserBase | undefined>(
```